akka/shopping-cart-scala/README.md

```diff
@@ -9,66 +9,60 @@ This is based on the sample code from [Akka Platform Guide](https://github.com/a
 - [grpcurl](https://github.com/fullstorydev/grpcurl#installation)
 - [Lightbend account and Bintray credentials](https://developer.lightbend.com/docs/telemetry/current/getting-started/start.html#lightbend-account-and-bintray-credentials)
 
-## How to run locally
+## Running the sample code
 
-1. Start a local Cassandra server on default port 9042 and a Kafka broker on port 9092. The included `docker-compose.yml` starts everything required for running locally.
+1. Start a local PostgresSQL server on default port 5432 and a Kafka broker on port 9092. The included `docker-compose.yml` starts everything required for running locally.
 
     ```shell
     docker-compose up -d
-    ```
-
-2. Create Cassandra keyspace and tables:
 
-    ```shell
-    # creates keyspace and all tables needed for Akka Persistence
+    # creates the tables needed for Akka Persistence
     # as well as the offset store table for Akka Projection
-    docker exec -i shopping-cart-service-scala_cassandra_1 cqlsh -t < ddl-scripts/create_tables.cql
-    ```
-
-    ```shell
+    docker exec -i shopping-cart-service_postgres-db_1 psql -U shopping-cart -t < ddl-scripts/create_tables.sql
+    
     # creates the user defined projection table.
-    docker exec -i shopping-cart-service-scala_cassandra_1 cqlsh -t < ddl-scripts/create_user_tables.cql
+    docker exec -i shopping-cart-service_postgres-db_1 psql -U shopping-cart -t < ddl-scripts/create_user_tables.sql
     ```
 
-3. Start a first node:
+2. Start a first node:
 
     ```shell
     sbt -Dconfig.resource=local1.conf run
     ```
 
-4. (Optional) Start another node with different ports:
+3. (Optional) Start another node with different ports:
 
     ```shell
     sbt -Dconfig.resource=local2.conf run
     ```
 
-5. (Optional) More can be started:
+4. (Optional) More can be started:
 
     ```shell
     sbt -Dconfig.resource=local3.conf run
     ```
 
-6. Check for service readiness
+5. Check for service readiness
 
     ```shell
     curl http://localhost:9101/ready
     ```
 
-7. Try it with [grpcurl](https://github.com/fullstorydev/grpcurl):
+6. Try it with [grpcurl](https://github.com/fullstorydev/grpcurl):
```

Comment:
I must be missing something.

I just checked https://github.com/akka/akka-platform-guide/blob/current/docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala/README.md and that doesn't contain the same as this file.

I was surprised to see `gatling.sbt` and that lead me into comparing this README too.

I don't think the `rsync` script is bring in what I expected. 🤔

Reply:
`gatling.sbt` was added here: #17.

I think one of us not seeing things correctly here. I've copied&pasted the file right now, and besides the sections at top and bottom, they are the same.

Reply:
Don't pay too much attention to what I'm saying. It's late where I live. 💤